pandoc-lua-engine/test/bytestring-reader.lua

```diff
@@ -1,4 +1,4 @@
-function ByteStringReader (input, opts)
+function ByteStringReader (input, _)
```

Comment:
Having reviewed and dealt with 100% of the lint errors thrown my `luacheck` on the entire code base, this is the one default lint I'm not so sure is helpful for this project. Since much of the Lua code in this repository is examples that people will be expected to extend rather than use as-is in production, the unused variable and unused argument lints might be counter productive. Thoughts?

Reply:
Using `_opts` might be another alternative.

Reply:
In my book that's a reasonable convention, and `luacheck` supports it. Some other Lua linters do not (e.g. `lua-language-server` flags it) but I'm still down. I just refactored all the lint handling using that method.